addon/components/hyper-table-v2/selection.hbs

```diff
@@ -0,0 +1,21 @@
+<div class={{this.classes}} {{did-insert this.setup}}>
+  <div class="count-container">
+    <i class="fas fa-users"></i>
+    {{#if this.isAllCreatorsSelected}}
```

Comment:
maybe just `allCreatorsSelected`
If multiple it shouldn't be "is" but rather "are" so the proper English would be "areAllCreatorsSelected" (Nit picking ofc ^^)
IMO, `allCreatorsSelected` is enough